src/sentry/static/sentry/app/api.tsx

```diff
@@ -453,11 +460,15 @@ export class Client {
         if (status !== 204 && !isStatus3XX) {
           try {
             responseJSON = await responseClone.json();
-          } catch {
-            // If the MIME type is `application/json` but decoding failed,
-            // this should be an error.
-            if (isResponseJSON) {
+          } catch (error) {
+            if (error.name === 'AbortError') {
               ok = false;
```

Comment:
Can you move `ok = false` out of these ifs?

Reply:
Hm... yeah that's true. Will add a different message for those cases so we can track it

Reply:
Look at this now, I take back what I said. If the response does not contain JSON then moving the `ok = false` out of the ifs will cause that to always be an error.